doc/requirements/eviction

```diff
@@ -0,0 +1,27 @@
+---
+group: Eviction
+---
+
+Eviction controlls a process of invalidating clean data from cache.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Eviction priorities
+id: priorities
+---
+
+The product shall shall perfrom eviction based on the priority information
+associated with IO classes.
+
+The product shall evict in following order:
+
+1. IO classes with excedeed occupancy limit.
+2. Lower priority IO classes (omit pinned IO Classes)
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Cold data eviction
+id: cold
+---
+
+The product shall evict the data within a single IO class in last recently used order.
```

Comment:
Adding the abbreviation is just a suggestion. You can skip that ;) Main change is last -> least ;)
```suggestion
The product shall evict the data within a single IO class in the least recently used (LRU) order.
```

Reply:
Typo still exists.